google/cloud/spanner/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@
 
 find_package(google_cloud_cpp_common CONFIG REQUIRED)
 find_package(google_cloud_cpp_grpc_utils CONFIG REQUIRED)
+find_package(bigtable_client CONFIG REQUIRED)
```

Comment:
Do we need this? I thought we didn't want spanner to depend on `bigtable_client`.

Reply:
I don't remember adding that myself. It may be an artifact from rebasing/merging.

Reply:
Ah, yes. I see the merge commits in your history. If you haven't already seen it the [instructions here](https://github.com/googleapis/google-cloud-cpp/blob/master/CONTRIBUTING.md#more-information-on-forks-and-pull-requests) should prevent that problem from happening.